pallets/gear/src/benchmarking/syscalls.rs

```diff
@@ -1609,4 +1611,25 @@ where
 
         Self::prepare_handle_with_const_payload(module)
     }
+
+    pub fn mem_grow(repetitions: u32, pages: u32) -> Result<Exec<T>, &'static str> {
```

Comment:
unfortunately this is wrong solution.

Ok, my solution:
1) For alloc:
1.1) Allocates 512 pages
1.2) Frees all 512 pages 
1.3) Allocate `pages` pages

2) For grow - I think you must check first that it will be really be linear function. Maybe it has bigger dependency on current memory size.